### PR TITLE
Azure BatchAI SDK updated to 2.0.0

### PR DESCRIPTION
**Pull request by llidev**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

SPEC for related API changes: https://github.com/Azure/azure-rest-api-specs/blob/master/specification/batchai/resource-manager/Microsoft.BatchAI/stable/2018-05-01/BatchAI.json

### CODE REVIEW

**Review by dsgouda — CHANGES_REQUESTED** [review]
Please regenerated the code using `generate.ps1` [here](https://github.com/lliimsft/azure-sdk-for-net/blob/48d45bfd4679fbafa75661877673602833cc2a43/src/SDKs/BatchAI/Management.BatchAI/generate.ps1) and commit changes to .txt file it generates
Please run `msbuild build.proj /t:build /p:Scopr=SDKs\BatchAI` and commit all changes
Please update the asssembly versions [here](https://github.com/lliimsft/azure-sdk-for-net/tree/48d45bfd4679fbafa75661877673602833cc2a43/src/SDKs/BatchAI/Management.BatchAI/Properties/AssemblyInfo.cs)

**Comment by dsgouda** [comment]
@lliimsft Please link the related spec

**Comment by lliimsft** [comment]
Spec was given in the description: https://github.com/Azure/azure-rest-api-specs/blob/master/specification/batchai/resource-manager/Microsoft.BatchAI/stable/2018-05-01/BatchAI.json

File: src/SDKs/BatchAI/Management.BatchAI/Generated/SdkInfo_BatchAI.cs [hunk 1]
```diff
@@ -1,22 +1,33 @@
...
+
+  internal static partial class SdkInfo
+  {
+      public static IEnumerable<Tuple<string, string, string>> ApiInfo_BatchAI
+      {
+          get
+          {
+              return new Tuple<string, string, string>[]
+              {
+                new Tuple<string, string, string>("BatchAI", "Clusters", "2018-05-01"),
```
Comment (dsgouda) [inline src/SDKs/BatchAI/Management.BatchAI/Generated/SdkInfo_BatchAI.cs]: Add the text `BatchAI_2018-05-01;` to the `AzureApiTag` node [here ](https://github.com/lliimsft/azure-sdk-for-net/blob/9c7155914cc93d38898227d6f3cb7bae5694ac15/src/SDKs/BatchAI/AzSdk.RP.props#L4)

**Review by dsgouda — CHANGES_REQUESTED** [review]
Apart from a minor comment looks great.

**Review by dsgouda — APPROVED** [review]
LGTM

**Pull request merged**